packages/harpguru-core/src/utils/get-renderable-tone-tuples/get-renderable-tone-tuples.ts

```diff
@@ -0,0 +1,32 @@
+import { isPitchId, isNaturalPitch, getPitch } from 'harpstrata'
+import type { PitchIds, DegreeIds } from 'harpstrata'
+
+import type { RenderableToneTuples } from '../../types'
+
+export const getRenderableToneTuples = (
+  sourceId: DegreeIds | PitchIds | undefined
+): RenderableToneTuples => {
+  if (sourceId === undefined) return [['', '']]
+
+  if (!isPitchId(sourceId)) {
+    const [note, ...modifiers] = sourceId.split('')
+
+    return [[note, modifiers.join('')]]
```

Comment:
This'll be changed when we have a tighter definition of `RenderableToneTuple` since the source values will be known to only have 1 or 2 characters.

Reply:
Raised as tech debt in https://github.com/js-jslog/harpguru/issues/6